pkg/manifest/internal/persistence/manifest_persitence.go

```diff
@@ -83,9 +84,33 @@ type Manifest struct {
 	Accounts          []Account `yaml:"accounts,omitempty"`
 }
 
+type AccountUUID struct {
+	Type  Type   `yaml:"type,omitempty" mapstructure:"type"`
+	Value string `yaml:"value" mapstructure:"value"`
+}
+
+// UnmarshalYAML Custom unmarshaler for AccountUUID able to parse simple shorthands (accountUUID: 1234) and full values.
+func (c *AccountUUID) UnmarshalYAML(unmarshal func(interface{}) error) error {
+	var data interface{}
```

Comment:
Huge nitpick and personal optiniom: `any` is nicer than `interface{}`

Reply:
I honestly have no feelings either way.. `any` is nice syntactic sugar, but if there's nothing else to change in this PR, I'd leave this as is.